client/dashboard/app/secondary-menu/index.tsx

```diff
@@ -48,14 +49,19 @@ function Help() {
 	const { user } = useAuth();
 	const { isLoading, isShown, setShowHelpCenter, setNavigateToRoute } = useHelpCenter();
 	const { recordTracksEvent } = useAnalytics();
-	const isMenuPanelEnabled = config.isEnabled( 'help-center-menu-panel' );
 	const [ helpCenterPage, setHelpCenterPage ] = useState( '' );
 
+	const [ isLoadingExperimentAssignment, experimentAssignment ] = useExperiment(
```

Comment:
Can we use `useMenuPanelExperiment` here too so all the assignations go through the same logic? Right now it's not clear why some places use  `useMenuPanelExperiment` and others use `useExperiment`.